_api-reference/security/authentication/auth-token.md

```diff
@@ -0,0 +1,94 @@
+---
```

Comment:
This API is not to be called directly and is used in the SAML login flow to vend a JWT after successful SAML authentication which will be used for the duration of the OSD session to authenticate with the backend OpenSearch. It technically is a security API, but I'm not sure if it needs public documentation or a note that it is an internal API.

The `/authinfo` endpoint is also primarily used by OpenSearch Dashboards to populate user information after logging in.

Reply:
Can Plugin Developer's interact with the API?

Reply:
not this API, its specific to the SAML authentication flow